.github/workflows/release-workflow.yaml

```diff
@@ -0,0 +1,28 @@
+name: release-workflow
+# This workflow is triggered on pushes to the repository.
+on:
+  push:
+    tags:
+    - '*'
+
+jobs:
+  publish:
+    name: publish-job
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: FranzDiebold/github-env-vars-action@v1.2.1
```

Comment:
Can we use the docker login github action here? 

Here is an example from one of my other repos
```
      - uses: actions/checkout@v2
      - name: docker build
        run: docker build -t slimm609/checksec:latest .
      - name: Login to DockerHub
        uses: docker/login-action@v1
        with:
          username: ${{ secrets.DOCKER_USERNAME }}
          password: ${{ secrets.DOCKER_PASSWORD }}
      - name: Push to Docker Hub
        uses: docker/build-push-action@v2
        with:
          push: true
          tags: slimm609/checksec:latest
```